gridt/src/app/core/models/identity.model.ts

```diff
@@ -0,0 +1,22 @@
+export interface Identity {
```

Comment:
Does it make sense to make this a [partial implementation](https://stackoverflow.com/a/57070274/1051438) of User?

Reply:
It does make sense. Although, if I want to do that I want to exclude the last_signal part. The Identity should be complete for new users and should not require some first interactions with a movement. Hence, in order to do this the User model should be split also. I don't think that is something we want to change now. Maybe an audit of the models in general can be beneficial to the code quality when the settings page is done.